drivers/hwinfo/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Kconfig -HW Info driver configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig HWINFO
+	bool "hwinfo driver"
```

Comment:
Maybe `"Hardware Information driver"`?  The lower case looks weird in `ninja menuconfig`.

Reply:
Done.